docs/features/notebooks.txt

```diff
@@ -161,8 +161,8 @@ In addition to the ``--config`` flag, configuration may also be supplied via a Y
 
 See :ref:`command-notebook-configuration` for details on the supported configuration options.
 
-Finally, to configure notebooks to run a predefined set of commands at startup, you can use a
-:ref:`startup hook <startup-hooks>` along with the ``--context`` option:
+Finally, to configure notebooks to run a predefined set of commands at startup, you can specify a
+:ref:`startup hook <startup-hooks>` with the ``--context`` option:
```

Comment:
I can see that the original phrasing was a little clumsy, though I think this new version has the issue that, in isolation, it makes it sound like `--context` is specifically for specifying startup hooks. (Which a reader will know is not the case if they also read the part about `--context` above, but that's kind of far away.) Maybe something like this?
```
you can include a :ref:`startup hook <startup-hooks>` in a directory specified with the ``--context`` option
```

Reply:
This is only to exercise the process but I'll make you're suggested change.